crates/matrix-sdk-crypto/src/types/qr_login.rs

```diff
@@ -327,6 +341,46 @@ mod test {
         );
     }
 
+    #[test]
+    fn parse_qr_data_reciprocate() {
+        let expected_curve_key =
+            Curve25519PublicKey::from_base64("2IZoarIZe3gOMAqdSiFHSAcA15KfOasxueUUNwJI7Ws")
+                .unwrap();
+
+        let expected_rendezvous =
+            Url::parse("https://rendezvous.lab.element.dev/e8da6355-550b-4a32-a193-1619d9830668")
+                .unwrap();
+
+        let data = QrCodeData::from_bytes(QR_CODE_DATA_RECIPROCATE)
+            .expect("We should be able to parse the QR code data");
+
+        assert_eq!(
+            expected_curve_key, data.public_key,
+            "The parsed public key should match the expected one"
+        );
+
+        assert_eq!(
+            expected_rendezvous, data.rendezvous_url,
+            "The parsed rendezvous URL should match to the expected one",
+        );
+
+        assert_eq!(
+            data.mode(),
+            QrCodeMode::Reciprocate,
+            "The mode in the test bytes vector should be Login"
```

Comment:
Is this a bad copypasto and this should read `should be Reciprocate`?

Reply:
Bad copypasta indeed :pinched_fingers:.